### PR TITLE
zh-cn: Format /web/javascript using Prettier

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats the `/web/javascript` folder (excluding `/web/javascript/reference` for now) of the Simplified Chinese locale using Prettier.

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
Thanks. Just something to consider/modify

File: files/zh-cn/web/javascript/guide/numbers_and_dates/index.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ JavaScript 最近添加了 {{jsxref("BigInt")}} 的支持，能够用于表示
...
 ```js
-1234567890
-42
+1234567890;
+42;
 // 以零开头的数字的注意事项：
-0888 // 888 将被当做十进制处理
-0777 // 在非严格格式下会被当做八进制处理 (用十进制表示就是 511)
+0888; // 888 将被当做十进制处理
+0777; // 在非严格格式下会被当做八进制处理 (用十进制表示就是 511)
```
[the remark this answers is not in the document]
Comment (queengooborg) [reply files/zh-cn/web/javascript/guide/numbers_and_dates/index.md:29]: This little section probably needs to be synchronized with upstream, WDYT?

[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/web/javascript/guide/numbers_and_dates/index.md:29]: Yeah. Please go ahead :)

[the remark this answers is not in the document]
Comment (queengooborg) [reply files/zh-cn/web/javascript/guide/numbers_and_dates/index.md:29]: Well, I don't speak Chinese, so I'm not actually able to do that... 🙃

**Review by yin1999 — APPROVED** [review]
LGTM. Thanks @queengooborg

**Pull request merged**